cmd/inspect.go

```diff
@@ -47,7 +48,7 @@ Partial certificates are also all printed for further inspection.
 				return err
 			}
 
-			analyser, err := analyse.NewAnalyser()
+			analyser, err := analyse.NewAnalyser(analyseOpts.MozillaRemovedCertsURL)
```

Comment:
Nit: Would it not make more sense in sending the whole `analyseopts` here and then having the Analyser utilise the exported fields? - that way adding additional opts is easier and you don't end up with a whole load of args to the NewAnalyser?